src/averageAges.js

```diff
@@ -55,10 +82,23 @@
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // 1. find a mother of each person (or only for men)
-  // 2. keep people who have mothers in the array
-  // 3. calculate the difference child.born - mother.born
-  // 4. return the average value
+  const parameter = onlyWithSon;
```

Comment:
Unnecessary variable 'parameter'. You could use 'onlyWithSon' directly in your conditional statement.